### ISSUE TITLE
Github_Org fails with API cycling update

### ISSUE THREAD

**Issue by domwhewell-sage** [blacklanternsecurity/bbot#1833]
**Describe the bug**
I noticed recently while using v2.1.0.4939rc0 the github org module goes into an error state
```python
2024-10-09 08:15:04,845 [WARNING] bbot.modules.github_org base.py:1470 Setting error state: Setting error state due to 82 failed HTTP requests
```
It was provided with a valid github api token, Im thinking its not setup to use the correct `self.helper` to send an api request and handle it correctly after the update

Example Logs:
```python
[DBUG] github_org: Handling ORG_STUB("Dell", module=TARGET, tags={'target', 'in-scope'})
[VERB] github_org: Validating whether the organization Dell is within our scope...
[DBUG] output.python: Got SCAN("{'id': 'SCAN:f71b63dfe1b548ec4f369c9cf710951088d64b2b', 'name': 'immense_judy', ...", module=TARGET, tags={'target', 'in-scope'}) from TARGET
[DBUG] output.python: SCAN("{'id': 'SCAN:f71b63dfe1b548ec4f369c9cf710951088d64b2b', 'name': 'immense_judy', ...", module=TARGET, tags={'target', 'in-scope'}) passed post-check
[DBUG] output.python: Got ORG_STUB("Dell", module=TARGET, tags={'target', 'in-scope'}) from TARGET
[DBUG] output.python: ORG_STUB("Dell", module=TARGET, tags={'target', 'in-scope'}) passed post-check
[VERB] github_org: Cycling API key
[VERB] github_org: Cycling API key
[VERB] github_org: Cycling API key
[VERB] github_org: Cycling API key
[VERB] github_org: Cycling API key
[VERB] github_org: Cycling API key
[VERB] github_org: Cycling API key
[VERB] github_org: Cycling API key
[VERB] github_org: Cycling API key
[VERB] github_org: Cycling API key
[VERB] github_org: Cycling API key
[VERB] github_org: Cycling API key
[VERB] github_org: Cycling API key
[VERB] github_org: Cycling API key
[VERB] github_org: Cycling API key
[VERB] github_org: Cycling API key
[VERB] github_org: Cycling API key
[VERB] github_org: Cycling API key
[VERB] github_org: Cycling API key
[VERB] github_org: Cycling API key
[VERB] github_org: Cycling API key
[VERB] github_org: Cycling API key
[VERB] github_org: Cycling API key
[VERB] github_org: Cycling API key
[VERB] github_org: Cycling API key
[VERB] github_org: Cycling API key
[VERB] github_org: Cycling API key
[VERB] github_org: Unable to validate that Dell is in-scope, skipping...
[DBUG] github_org: Finished handling ORG_STUB("Dell", module=TARGET, tags={'target', 'in-scope'})
```

**Comment by domwhewell-sage** [comment]
Ah I added some debug output to the `api_request()` function
```python
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token 8'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token 3'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token V'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token _'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token i'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token e'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token W'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token G'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token n'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token q'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token A'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token I'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token m'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token C'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token j'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token 9'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token d'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token b'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token v'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token 7'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token p'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token s'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token h'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token x'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token Q'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token Z'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token t'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token 1'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token g'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: {'headers': {'Authorization': 'token N'}, 'url': 'https://api.github.com/orgs/Dell'}
[VERB] github_org: Cycling API key
[DBUG] github_org: <Response [401 Unauthorized]>
[VERB] github_org: Unable to validate that Dell is in-scope, skipping...
```
Note: This api key has been changed incase anyone gets any ideas 😉

So it seems to by cycling over each letter of the API key. I am still supplying API keys in the old format but looks like thats still ok from the [README](https://github.com/blacklanternsecurity/bbot/blob/dev/README.md#api-keys)

It still happens if I supply the API key in a list format
```yaml
modules:
  github_org:
    api_key:
      - 21a270d5f59c9b05813a72bb41707266
```

**Comment by TheTechromancer** [comment]
Oh no. I can't believe the tests missed this

**Comment by domwhewell-sage** [comment]
I suspect it is because the tests dont validate the "Authorization: Bearer" header and just respond with the content if the correct URL is requested

**Comment by TheTechromancer** [comment]
Fixed in d6200bee.
